### DISCUSSION TITLE
SSL Verification for Webhook urls

### DISCUSSION THREAD

**Discussion by NusretOzates** [chatwoot/chatwoot#4829] [General]
Hi!

I have a chatwoot instance in our openshift platform, created an API inbox, and gave an HTTPS webhook URL but its certificate is self-signed and will be like that because it is only accessible inside the server. I couldn't find a env variable or a setting to close ssl verification. Do we have one?

**Comment by sojan-official** [comment]
@NusretOzates at the moment self-signed certificates are not supported. if it's only accessible inside the server, why not try connecting over HTTP version of the URL ?

**Comment by NusretOzates** [comment]
Unfortunately, I don't have the authority to decide that but I will see what can I do. Will it be supported one day?

**Reply by sojan-official** [reply]
@NusretOzates could be in the future. but it's not an immediate priority for us right now.
